tests/main/interfaces-calendar-service/task.yaml

```diff
@@ -35,7 +35,7 @@ restore: |
     if pid="$(pidof gvfsd-metadata)"; then
         kill -9 "$pid" || true
     fi
-    rm -rf "$XDG"
+    retry "rm -rf $XDG"
```

Comment:
With the changes I suggested above this would become: `retry rm -rf "$XDG"`

Reply:
done